third_party/matplotlibcpp.h

```diff
@@ -52,6 +52,8 @@ struct _interpreter {
     PyObject *s_python_function_axis;
     PyObject *s_python_function_xlabel;
     PyObject *s_python_function_ylabel;
+    PyObject *s_python_function_xticks;
```

Comment:
We should probably be nice OSSers and pull request some of these changes into their repo.

Reply:
I've put in a PR

Reply:
:+1: